app/controllers/works_controller.rb

```diff
@@ -306,7 +306,7 @@ def create
       render :new && return
     elsif params[:cancel_button]
       flash[:notice] = ts('New work posting canceled.')
-      redirect_to current_user && return
+      redirect_to current_user and return
```

Comment:
No change needed, just noting that we use `redirect_to(current_user) && return` in some places to get around this warning.